src/reference-manual/types.Rmd

```diff
@@ -1269,7 +1280,7 @@ int N = 5;
 declares the variable `N` to be an integer scalar type and at the
 same time defines it to be the value of the expression `5`.
 
-### Assignment typing {-}
+#### Assignment typing {-}
```

Comment:
H2s are put on their own pages (unless there is something I can do to disable this), so pulling everything up a level will split this in three -- is that fine? The H4 looks fine in HTML, but in latex it ends up in italics rather than bold